main.cpp

```diff
@@ -73,12 +25,27 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void ToString() {
+    cout<<"Number to Color Mapping"<<endl;
```

Comment:
ToString is meant to place the contents of the color-code reference in a string, not on the console.